packages/notifi-react-card/lib/components/intercom/NotifiIntercomCardContainer.tsx

```diff
@@ -73,29 +79,46 @@ export const NotifiIntercomCardContainer: React.FC<
     <div
       className={clsx(
         darkMode ? 'notifi__dark' : 'notifi__light',
-        'NotifiIntercomCard__container',
+        startChat
+          ? 'NotifiIntercomCard__chatWindowContainer'
+          : 'NotifiIntercomCard__container',
         classNames?.container,
       )}
     >
-      <h1 className={clsx('NotifiIntercomCard__title', classNames?.title)}>
-        {companySupportTitle}
-      </h1>
-      <div
-        className={clsx('NotifiIntercomCard__subtitle1', classNames?.subtitle1)}
-      >
-        {companySupportSubtitle}
-      </div>
-      <div
-        className={clsx('NotifiIntercomCard__subtitle2', classNames?.subtitle2)}
-      >
-        {companySupportDescription}
-      </div>
-      {children}
-      {contents}
-      <NotifiStartChatButton
-        disabled={disabled}
-        classNames={classNames?.NotifiStartChatButton}
-      />
+      {startChat ? (
```

Comment:
create NotifiConsumerFTU, and showing NotifiConsumerFTU when stats is 'fetched'. and moved all the logic for showing startChatView in the NotifiConsumerFTU.

Reply:
I think how you have it now works as well. 

I was thinking something more along the lines of where each page or 'view' is tied to its own state. 

```
useEffect(() => {
  if (userHasConversationActive) 
     setEditState('conversationActive');
})

case 'fetched': 
   return StartChatView;
case 'conversationActive'
  return ChatWindowView;
```

You will find this helpful when you create the Alert Settings page so that you don't have a nested conditional on what you render.

Maybe look into: `packages/notifi-react-card/lib/components/subscription/SubscriptionCardV1.tsx`

Reply:
updated